pkg/api/const.go

```diff
@@ -483,4 +483,8 @@ const (
 	ClientCertificateAuthMethod = "client_certificate"
 )
 
-const TLSStrongCipherSuites = "TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305,TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA,TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA,TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA,TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA"
+// TLSStrongCipherSuitesAPIServer is a kube-bench-recommended allowed cipher suites for apiserver
+const TLSStrongCipherSuitesAPIServer = "TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305,TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA,TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA,TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA,TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA"
```

Comment:
Because the allowed list is different for apiserver and kubelet, renamed the pre-existing const to reflect that it is for apiserver only.

Reply:
where is this copied from? dont see it https://github.com/aquasecurity/kube-bench/blob/master/cfg/1.13/master.yaml#L505

Reply:
is the kubelet cipher suites allow list that this PR delivers correct?

if we are being too permissive w/ apiserver I would advocate we make that change in a separate PR, the change above is just to the const name

Reply:
Re-opened #680 to apply kube-bench-recommended apiserver cipher suites values.

Verified that kube-bench kubelet recommendations match the changes in this PR.